packages/ui/src/assets/css/page.css

```diff
@@ -1,12 +1,17 @@
+:root {
+  --page-gap: var(--dz-page-gap);
+}
+
```

Comment:
you don't need to do this here. Instead you can add the local variable under .page class
```
.page {
  --_page-gap: var(--dz-page-gap);
}

gap: var(--_page-gap);

```